data-prepper-benchmarks/mapdb-benchmarks/build.gradle

```diff
@@ -24,5 +24,12 @@ repositories {
 }
 
 dependencies {
+    implementation project(':data-prepper-api')
     implementation project(':data-prepper-plugins:mapdb-prepper-state')
 }
+
+checkstyle {
+    checkstyleMain.enabled = false
+    checkstyleTest.enabled = false
+    checkstyleJmh.enabled = false
+}
```

Comment:
I know why we turned it off, but I would like to have everything conform to checkstyle.